loopchain/crypto/signature.py

```diff
@@ -17,16 +17,10 @@
 import getpass
 import hashlib
 import logging
-<<<<<<< HEAD
-from typing import Union
-
-=======
 from typing import Union, Type, TypeVar
->>>>>>> 9415a880... [LC-339] Improve type hint
```

Comment:
It would be conflict with #224 (https://github.com/icon-project/loopchain/pull/224/commits/84efffdbb0d48c9dc1c00ccb9da587d828d3e91a) probably.